### PR TITLE
rest API: organisation filter

### PR DESCRIPTION
**Pull request by jma**
* NEW Filter list result for REST API by organisation.

Signed-off-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

### CODE REVIEW

File: rero_ils/modules/items/listener.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Arguments:
+    - ``json``: The dumped record dictionary which can be modified.
+    - ``record``: The record being indexed.
+    - ``index``: The index in which the record will be indexed.
+    - ``doc_type``: The doc_type for the record.
+    """
+    # TODO: this multiply the indexing time by 5, try an other way!
+    item_index_name = ItemsSearch.Meta.index
+    if index.startswith(item_index_name):
+        org_pid = record.get_library().replace_refs()['organisation']['pid']
```
Comment (BadrAly) [inline rero_ils/modules/items/listener.py]: This line fails when running `setup.sh`:
```
  File "/Users/baa/devel/rero/rero-ils/rero_ils/modules/items/listener.py", line 43, in enrich_item_data
    org_pid = record.get_library().replace_refs()['organisation']['pid']
AttributeError: 'Record' object has no attribute 'get_library'
```

**Comment by BadrAly** [comment]
UI: Resources are properly filtered by logged users organisation  👍

File: rero_ils/modules/items/listener.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# You should have received a copy of the GNU General Public License
+# along with RERO ILS; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""."""
```
Comment (BadrAly) [inline rero_ils/modules/items/listener.py]: docs.

File: rero_ils/modules/locations/listener.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# You should have received a copy of the GNU General Public License
+# along with RERO ILS; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""."""
```
Comment (BadrAly) [inline rero_ils/modules/locations/listener.py]: docs

File: rero_ils/modules/items/listener.py (hunk 1 above)
Comment (jma) [reply rero_ils/modules/items/listener.py]: Solved

**Pull request merged**